src/theory/quantifiers/sygus/cegis_unif.cpp

```diff
@@ -83,6 +116,69 @@ bool CegisUnif::constructCandidates(const std::vector<Node>& enums,
                                     std::vector<Node>& candidate_values,
                                     std::vector<Node>& lems)
 {
+  NodeManager* nm = NodeManager::currentNM();
+  if (d_no_unif)
+  {
+    candidate_values.insert(
+        candidate_values.end(), enum_values.begin(), enum_values.end());
+    if (options::sygusDirectEval())
```

Comment:
You should generally never copy and paste code.  The point of making this class inherit from cegis is so that this code can be encapsulated in a function that is called in both places.